lib/Database/Database.cpp

```diff
@@ -471,7 +471,6 @@ Optional<GlobalSymbolKind> db::getGlobalSymbolKind(SymbolKind K) {
     case SymbolKind::Constructor:
     case SymbolKind::Destructor:
     case SymbolKind::ConversionFunction:
-      return None;
```

Comment:
This doesn't look right - it would fall through to `GlobalSymbolKind::Enum`.

Reply:
Whoooops. Thanks

Reply:
I would have just yanked both lines and then moved them 😅